backend/api/constants.py

```diff
@@ -9,20 +9,25 @@
 src_dir = pathlib.Path(__file__).parent.resolve()
 
 PROJECT_EXPIRE_AFTER = datetime.timedelta(days=7)
-if not os.getenv("POSTGRES_URI"):
-    msg = "Please set the POSTGRES_URI environment variable"
-    raise OSError(msg)
 
 logger = logging.getLogger(src_dir.name)
 
 
+def postgres_uri():
+    uri = os.getenv("POSTGRES_URI")
+    if not uri:
+        msg = "Please set the POSTGRES_URI environment variable"
+        raise OSError(msg)
+    return uri
+
+
 @dataclass
 class BackendConf:
     """
     Backend configuration, read from environment variables and set default values.
     """
 
-    postgres_uri = os.getenv("POSTGRES_URI")
+    postgres_uri = postgres_uri()
```

Comment:
This reduces the value of BackendConf and offers two ways to retrieve it. Keep it as it was.